docs/team/mep.md

```diff
@@ -0,0 +1,265 @@
+MMTk Enhancement Proposal (MEP)
+===============================
+
+An MMTk Enhancement Proposal (MEP) is a more formal variant of issue.  It has a special format, and
+will undergo a more thorough review process.  Its goal is helping the MMTk developers making more
+informed decisions.
+
+MEP is inspired by the Java Enhancement Proposal, described in https://openjdk.org/jeps/1  However,
+unlike JEP which is more open-ended, MEP is more focused on making design decisions.
+
+# When is MEP required?
+
+An MEP is required when making a significant change to the design of the MMTk core.  It is
+applicable to any kind of significant changes, including but not limited to:
+
+-   Bug fixes, including performance bug fixes, that require change to a major part of the MMTk
+    core.
+-   Changes to the MMTk core to implement a feature demanded by bindings.
+-   Major refactoring to the MMTk core.
+
+One purpose of MEP is reducing the risks to the future development of MMTk.  Large-scale changes and
```

Comment:
I replaced this sentence with what Steve wrote in the "risks" section.  "The core purpose of the MEP process is to avoid the unintentional introduction of changes that bring long-term negative impacts to MMTk." Saying it is the "core purpose" sounds better than "one purpose is...", but it still doesn't rule out the possibility that there are other purposes.

I think we don't need to emphasize whether it is the only purpose or not.  If we find MEP may serve other purposes, we can update this document.

> I think it is a good idea to state what 'risks' means...

We now have a dedicated "risks" section to discuss the details of the risks.

Reply:
Personally, I think MEP is also useful if I want to make a major change to MMTk, but I don't know exactly how to implement it.  The MEP process should be able to help by allowing members to discuss the best implementation strategy together.  I'd really like the MEP process to help with the PR https://github.com/mmtk/mmtk-core/pull/1004 because it is important but I haven't figured out the satisfactory implementation, yet.  We can try that later and see if MEP can serve this purpose.